app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+class Animal:
+    alive = list()
```

Comment:
The `alive` list is a class variable shared among all instances of `Animal`. This could lead to unexpected behavior, especially in multi-threaded environments or when instances are frequently created and destroyed. Consider using an instance variable if this is not the intended design.